<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>791</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QPushButton" name="matrix_button">
         <property name="text">
          <string>Gerar uma Matriz em Excel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="matrix_colors_button">
         <property name="text">
          <string>Cores da Imagem</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="font">
        <font>
         <pointsize>29</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="contrast_button">
         <property name="text">
          <string>Ajuste de Contraste</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="brightness_button">
         <property name="text">
          <string>Ajuste de Brilho</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="difference_button">
         <property name="text">
          <string>Diferen√ßa entre Imagens</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="histogram_button">
         <property name="text">
          <string>Transferir Histograma</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="mirror_button">
         <property name="text">
          <string>Espelhar Imagem</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rotate_horizontally_button">
         <property name="text">
          <string>Girar Imagem na Horizontal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rotate_vertically_button">
         <property name="text">
          <string>Girar Imagem na Vertical</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scale_button">
         <property name="text">
          <string>Mudar Escala</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>590</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="leave_button">
       <property name="text">
        <string>Sair</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_image_button">
       <property name="text">
        <string>Carregar Imagem</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Preview</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>791</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1172</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
